api/helpers/sdtd/validate-item-name.js

```diff
@@ -1,5 +1,7 @@
 var sevenDays = require('machinepack-7daystodiewebapi');
 
+const SdtdApi = require('7daystodie-api-wrapper')
```

Comment:
```suggestions
```

Reply:
Both can go since I use the sails helper now. Good catch!